devops/actions/cached_checkout/action.yml

```diff
@@ -10,7 +10,7 @@ inputs:
     description: |
       Commit-ish to merge post-checkout if non-empty. Must be reachable from the
       default_branch input paramter.
-    default: 'FETCH_HEAD'
```

Comment:
the only place where we `merge_commit` is directly used in all of CI is in cached checkout?

Reply:
Yes.